blueocean-web/src/main/java/io/jenkins/blueocean/dev/RunBundleWatches.java

```diff
@@ -337,6 +337,8 @@ public void destroy() {
                             }
                         }
 
+                        final File watchDir = new File(projectDir, "src");
+
```

Comment:
Might wanna move the `.less` files to `/src` within the JDL too.

Reply:
Right, might be worthwhile. Having a consistent project structure is a Good Thing&trade;.

Reply:
but for mergeability, I don't want to move until after this lands in master

Reply:
Good call